sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -39,8 +39,12 @@
 from numba.errors import TypingError
 from sdc.hiframes.pd_dataframe_type import DataFrameType
 
+from sdc.datatypes.hpat_pandas_dataframe_rolling_types import _hpat_pandas_df_rolling_init
+from sdc.datatypes.hpat_pandas_rolling_types import (
+    gen_sdc_pandas_rolling, sdc_pandas_rolling_docstring_tmpl)
 from sdc.datatypes.hpat_pandas_series_functions import TypeChecker
```

Comment:
Hasn't TypeChecker been moved to common_functions?

Reply:
It has been, but accidentally it has been copied back by somebody. Anyway I didn't change the code in this PR :)